staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2110,6 +2137,7 @@ func DescribePodTemplate(template *corev1.PodTemplateSpec, w PrefixWriter) {
 	}
 	describeContainers("Containers", template.Spec.Containers, nil, nil, w, "  ")
 	describeVolumes(template.Spec.Volumes, w, "  ")
+	describeTopologySpreadConstraints(template.Spec.TopologySpreadConstraints, w, "  ")
```

Comment:
Strange we don't show all the information for a pod template.  I wonder what the criteria is for deciding what should be shown for a pod template.  For example, we don't show tolerations.

I think it is fine to add this here.  I wonder if we should (as a separate PR) add the other spec fields, where they make sense.

Reply:
> I think it is fine to add this here. I wonder if we should (as a separate PR) add the other spec fields, where they make sense.

I have the same question in my mind. As far as I understand, since `describe` is _nicer_ way to showing resources, there is no promise that `describe` shows everything `get` shows.